### ISSUE TITLE
UPM Package Description for Android/IOS

### ISSUE THREAD

**Issue by AndrewKahr** [PlayEveryWare/eos_plugin_for_unity#146]
**Is your feature request related to a problem? Please describe.**
I'm looking to build the development branch with iOS and Android support, however it doesn't look like iOS specific code/plugins are being copied into the built UPM. I'm basing this off the fact that no iOS/Android folders are created in the runtime or editor folder of the plugin nor are they entries under the EOS config editor. I tried manually adding iOS/Android-related files to the package description json however I'm not sure of the exact configuration so I'm getting errors based on the location of the copied files and possible missing files.

**Describe the solution you'd like**
Is there a ready-made json config with the necessary files already setup inside that I could use to build the package to test with?

**Describe alternatives you've considered**
Attempted to manually add the necessary files to the json, however, I don't fully understand the layout or what files do/don't need to be copied over.

**Comment by andrew-hirata-playeveryware** [comment]
That is correct that they are not! I'm actually working on getting this ready so that it will integrate cleanly into the UPM today.

**Comment by AndrewKahr** [comment]
Thanks so much for the info!
